spec/openapi.yaml

```diff
@@ -58,22 +58,32 @@ paths:
 
   /api/v1/files/{storage}/{path}:
     parameters:
+      - in: header
+        name: Accept-Language
+        description: Defines a language of the response
+        schema:
+          type: string
+          example: "cs"
+      - in: header
+        name: Accept
+        description: Preferred content-type of response - application/json or text/html, all other are returned as text/plain
+        schema:
+          type: string
+          default: text/plain
       - in: path
         name: storage
-        required: true
         description: The target storage
+        required: true
         schema:
           type: string
-          enum:
-            - local
-            - sdcard
+          example: "/local"
       - in: path
         name: path
-        required: true
         description: Path to the file
+        required: true
         schema:
           type: string
-          example: "PrusaLink gcodes/Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+          example: "/local/Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
```

Comment:
This one is probably bit technical, but if we say that the example of `path` is `/local/something`, then the whole URL would be `/local//local/...`. I think the original example was correct.